lib/td/client/api/job.rb

```diff
@@ -165,7 +165,7 @@ def job_result_each(job_id, &block)
     upkr = MessagePack::Unpacker.new
     infl = nil
 
-    get("/v3/job/result/#{e job_id}", {'format'=>'msgpack'}) {|res, chunk, current_total_chunk_size|
+    get("/v3/job/result/#{e job_id}", {'format'=>'msgpack'}, {:resume => true}) {|res, chunk, current_total_chunk_size|
       if res.code != 200
```

Comment:
Why I added {:resume => true} is mainly worried the case when get() is passed a block, we don't know what the block handles the arguments.

As you say it can resume by default if it if its block argument is in td-client-ruby in ruby and considered to be safe.